tests/unit/slider/slider_methods.js

```diff
@@ -98,6 +98,17 @@ test( "value", function() {
 	});
 	element.slider( "option", "value", 2.4 );
 	equal( element.slider( "value" ), 2.4, "value is set to max with 0.01 step" );
+
+	element = $("<div></div>").slider({
+		value:100,
+		min: 10,
+		max: 500,
+		step: 50
+	});
+
+	//set value with min that less than step and more than zero
```

Comment:
This comment and the assertion message provide the same value, drop the comment.